City-of-Helsinki/berth-reservations-admin
# VEN-949, VEN-1066 | Resend invoice that has been already sent

## Description :sparkles:

* Resend invoices that have already been sent, with a new due date. Implemented in:
  * Application list (in multi-invoicing)
  * Application view (in offer card)
  * Customer view (in open invoices card)
* Changed behavior so that completing the offer page returns to the application and back button returns to the application list

## Issues :bug:

### Closes :no_good_woman:

* [VEN-949](https://helsinkisolutionoffice.atlassian.net/browse/VEN-949): FE: Updating duedate already sent invoices
* [VEN-1066](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1066): Resend invoice that has been already sent

## Testing :alembic:

### Automated tests :gear:️

* Updated existing tests

### Manual testing :construction_worker_man:

* Resending an invoice should be possible via the three views listed in the description
* The due date should change for the invoice - confirm it in the customer view which shows the due date

## Screenshots :camera_flash:

![image](https://user-images.githubusercontent.com/15479131/107527794-d321cd00-6bc1-11eb-95c7-d40e51789f28.png)
![image](https://user-images.githubusercontent.com/15479131/107527937-fc425d80-6bc1-11eb-8348-15bf3aef64df.png)
![image](https://user-images.githubusercontent.com/15479131/107528019-0ebc9700-6bc2-11eb-89b3-d68d89586cb5.png)
![image](https://user-images.githubusercontent.com/15479131/107528120-298f0b80-6bc2-11eb-9679-061520afe7c8.png)
